### ISSUE TITLE
Meaning of the video icons in playlists is not always apparent

### ISSUE THREAD

**Issue by Sturek** [elan-ev/studip-opencast-plugin#763]
![icons_oc](https://github.com/elan-ev/studip-opencast-plugin/assets/6900803/f83424bc-7956-4161-ad69-16410a18cf13)

As there is no legend, it is not always clear what each icon means. This can be confusing, as some icons also invite different interpretations.

**Comment by Sturek** [comment]
Perhaps this will be solved by resolving #815

**Comment by rlucke** [comment]
Status icons also have a title attribute, which makes it clear on mouseover what is going on with the video. Unfortunately, the current solution is not barrier-free. It lacks role=“img” and aria-label on the span and alt=“” on the img.
